JDI.Light/JDI.Light/Elements/Composite/FileInput.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
top using sections are not required

![image](https://user-images.githubusercontent.com/6689884/54680372-bc2fe880-4b1a-11e9-9ec2-ed35bbc3a8b9.png)